source/common/http/conn_manager_config.h

```diff
@@ -17,6 +17,7 @@ namespace Http {
   COUNTER  (downstream_cx_total)                                                                   \
   COUNTER  (downstream_cx_ssl_total)                                                               \
   COUNTER  (downstream_cx_http1_total)                                                             \
+  COUNTER  (downstream_cx_upgrades_total)                                                         \
```

Comment:
I'm a little concerned this might be confusing for people looking for WS socket stats? Probably not actionable right now but something to think about. Not sure if folks will find it confusing or not. We might want to have more docs on this? Feel free to do in your other removal PR if you are inclined.

Reply:
SGTM